src/variables.jl

```diff
@@ -351,13 +345,12 @@ function set_lower_bound(v::VariableRef, lower::Number)
     newset = MOI.GreaterThan(convert(Float64, lower))
     # do we have a lower bound already?
     if has_lower_bound(v)
-        cindex = lower_bound_index(v)
+        cindex = _lower_bound_index(v)
         MOI.set(backend(owner_model(v)), MOI.ConstraintSet(), cindex, newset)
     else
         @assert !is_fixed(v)
-        cindex = MOI.add_constraint(
+        MOI.add_constraint(
```

Comment:
We should have a shortcut in `JuMP.add_variable` to skip the branching on `has_lower_bound` and directly go in the `else`, e.g., by creating an auxiliary function for this `else`